nodestream/project/pipeline_scope.py

```diff
@@ -1,15 +1,18 @@
 from typing import Dict, Iterable, List
 
-from ..exceptions import MissingExpectedPipelineError
 from ..file_io import LoadsFromYaml, SavesToYaml
-from ..model import (
+from ..schema.schema import (
     AggregatedIntrospectiveIngestionComponent,
     IntrospectiveIngestionComponent,
 )
 from .pipeline_definition import PipelineDefinition
 from .run_request import RunRequest
 
 
+class MissingExpectedPipelineError(ValueError):
+    pass
```

Comment:
Is this intended to be a pass?

Reply:
Yeah just providing a specific type for the error .